cmd/centre/main.go

```diff
@@ -317,11 +318,18 @@ func main() {
 
 	if *inf != "" {
 		centre, _, err := lookupIP(*hostFile, "centre")
+		var ip net.IP
 		if err != nil {
 			log.Printf("No centre entry found via LookupIP: not serving DHCP")
-		} else if *ipv4 {
-			ip := centre.To4()
+		}
+		if *ipv4 {
+			if err == nil {
```

Comment:
This is a bit subtle and may confuse people later. But if it works I guess it's ok.